### DISCUSSION TITLE
Improve JSON field validation

### DISCUSSION THREAD

**Discussion by bamboowonder** [directus/directus#12863] [Feature Requests]
### Describe the Bug

When a json field is configured in UI, if you incorrectly edit the json, there is a small visual indication of a syntax error.

However if that is ignored, and you hit save anyway, the changes are silently dropped and reverted to the existing value.
Assuming the user doesn't notice the tiny exclamation mark warning, they may assume the changes were saved rather than rejected.

Ideally, there would be an option to either revert to valid json, or continue editing until the json passes validation.

### To Reproduce

in postgres, create a jsonb column
in directus, Schema Type = json,  

It should be noted, that the Save button isn't activated unless another field is also changed.  The other field will save but the json will be reverted.

However, when the json field is part of a record opened on the side bar as a foreign field lookup, the save button isn't deactivated with just the json being edited and invalid.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
